src/CLI/CLI.php

```diff
@@ -278,7 +278,7 @@ public function match(): ?Task
      * Get runtime params for the provided Hook
      *
      * @param Hook $hook
-     * @return array
+     * @return array<string>
```

Comment:
I don't think this is supposed to be `array<string>` considering how `$params` is used.